patterns/core/Header/User/LoginForm.js

```diff
@@ -0,0 +1,52 @@
+import { useState } from 'react'
+
+import { Form } from './LoginBox'
+import { FormField, FormInput, FormStatus, Button } from '../../..'
+import { useTranslation } from '../../../../utils'
+
+const dummyLoginUser = async (data) => {
+  return new Promise((res) => setTimeout(() => res(data), 1000))
+}
+
+const LoginForm = ({ onPasswordForgetSwitch }) => {
+  const { t } = useTranslation()
+
+  const [formStatus, setFormStatus] = useState('')
+  const [isFormLoading, setFormLoading] = useState(false)
+
+  const onSubmit = async (data) => {
+    setFormLoading(true)
+
+    const success = await dummyLoginUser(data)
+
+    setFormStatus(success ? 'success' : 'error')
+    setFormLoading(false)
+  }
+
+  return (
+    <Form
+      title={t('LOGIN_TITLE')}
+      buttonText={t('LOGIN_BUTTON')}
+      formId="login-box__form"
+      onSubmit={onSubmit}
+      loading={isFormLoading}
+    >
+      <FormStatus errorMessage={t('LOGIN_ERROR')} status={formStatus} />
```

Comment:
I did not add the `successMessage` prop here because after the user logs, the "logged-in"-view is directly displayed. So the `successMessage` would never be shown to the user.

Reply:
okay. good point. In this case its sufficient. Generally giving the user an positive feedback for an successful action will increase his learning curve